ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/PersistentQueryCleanupImpl.java

```diff
@@ -47,20 +48,23 @@ public void cleanupLeakedQueries(final List<PersistentQueryMetadata> persistentQ
         .stream()
         .map(PersistentQueryMetadata::getQueryApplicationId)
         .collect(Collectors.toSet());
-    try {
-      final Set<String> allStateStores = new HashSet<>(Arrays.asList(new File(stateDir).list()));
```

Comment:
@lct45 Some side cleanup -- we should just handle `null` explicitly instead of handling an exception.